randomised_quicksort.cpp

```diff
@@ -0,0 +1,59 @@
+#include<iostream>
+#include <bits/stdc++.h>
+#include<stdlib.h>
+#include<time.h>
+using namespace std;
+int partition(int *a,int start,int end)
+{
+    int pindex=0;
+    int pivot=a[end];
+    for(int i=0;i<=end-1;i++)
+    {
+        if(a[i]<=pivot)
+        {
+            swap(a[i],a[pindex]);
+            pindex=pindex+1;
+        }
+    }
+    swap(a[pindex],a[end]);
+    return pindex;
+}
+int random_partition(int *a,int start,int end)  #it will choose the random element as pivot element
```

Comment:
`#` is used for single line comments in **Python**. please replace `#` with `\\`